tests/test_image_to_text_example.py

```diff
@@ -16,6 +16,9 @@
         "bf16": [
             ("llava-hf/llava-1.5-7b-hf", 1, 87.2901500056982),
         ],
+        "fp8": [
+            ("llava-hf/llava-1.5-7b-hf", 1, 123.00953973789325),
```

Comment:
i got 2x performance for both bf16 and fp8 using docker 1.16.0-447. can you confirm?

Reply:
this was tested on 1.16.0-388
I'll test also on your version

Reply:
It seems latest available on Gaudi devcloud is 1.16.0-410

Reply:
@schoi-habana can you please try to run your test using **--max_new_tokens 20** ? If I remove it, so it uses its default value 100, then the Throughput is 2x as you pointed out.

Reply:
right i was not using --max_new_tokens 20. i got about same performance as you with --max_new_tokens 20